### ISSUE TITLE
Dragging in a comment to select text also scrolls the block editor sideways

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap#2660]

**Comment by Daniel4-Scratch** [comment]
I don't really get this.

**Comment by jmoenig** [comment]
yep, confirmed. Workaround: First click into the text so a caret appears, then press the mouse again and hold it down to select text. This will avoid dragging the scripting area. Eventually we'll get around to come up with something better.
